docs/source/configuration/slots.md

```diff
@@ -0,0 +1,89 @@
+# Slots
+
+Slots are insertion points in the Volto rendering tree structure.
+This concept is inherited from Plone ClassicUI artifact `Viewlets`.
+
+## Anatomy
+
+Slots are named, and they can contain a list of different slot components.
+Slots components are also named, and they are registered in the configuration registry using a specific API for slots.
+The main trait of a slot component is that its render are controlled by a list of conditions (predicates).
+Multiple slot components can be registered under the same name, as long as they have different predicates.
+
+Slot (eg. `toolbar`)
+  - SlotComponent (eg. `edit`)
+    - predicates (eg. only appear in `/de/about`)
+    - predicates (eg. only appear if content type is one of `Document` or `News Item`)
+    - no predicates (eg. always appear)
+  - SlotComponent (eg. `contents`)
+  - SlotComponent (eg. `more`)
```

Comment:
I'm not clear of the concept that this markdown is supposed to represent. Is this accurate?

1. This is a tree structure, where the root is `Slot`. Assuming that is correct, `Slot` should be inside the nested list, not as a separate paragraph.
2. Then there are 3 `SlotComponent` thingies, each with their own name.
3. Within the `edit` `SlotComponent` thingie, there are 2 lists of predicates and a default condition for when none of the predicates return `true`.

Please let me know.

Reply:
Yes, it's like this.

Reply:
@sneridagh in this example, can you clarify what should happen with the `edit` slot component when both of its predicates return `false`? Originally you said "always appear", but then why would we need predicates for this slot component? Should it instead be "do not display"?

Reply:
See suggested revision in c4cfe86ce3aacfd829cde31ea868f7f09026d855